### DISCUSSION TITLE
Snapshot during time domain simulation

### DISCUSSION THREAD

**Discussion by diodezener** [CURENT/andes#203] [General]
Hi Dr. Cui,

I'm wondering is it possible to save a "snapshot" of the system during time domain simulation like in PSCAD or PSS/E, which allows a rerun later from the given snapshot in time?

**Comment by cuihantao** [comment]
Hi @diodezener ,

It is not yet supported but is a desired feature. I will target v1.6.0 to have this feature ready.

The current challenge is that the discrete flags (limiter status, etc) and memory units need to be properly externalized.

**Reply by diodezener** [reply]
Thanks for the reply. Looking forward to it.
